app/controllers/events_controller.rb

```diff
@@ -102,6 +103,13 @@ def set_event
       @event = Event.find(params[:id])
     end
 
+    def set_playercount_per_team
+      if event_params[:player_type] == "single" || @event.type == 'Rankinglist'
```

Comment:
You could move this to set_associations, since it would make more sense to be integrated there (in the end this is an association as well). Also you could use `@event.player_type == Event.player_types[:single]` there instead of the `event_params` stuff. Or you get rid of this entirely and set a default value in the database for `min_players_per_team` and `max_players_per_team`, which would simplify some other things like the factories as well.